### PR TITLE
[Tests-Only] Enable codeStyle

### PR DESCRIPTION
**Pull request by jasson99**
## Description:
This pr :
- [x] enables codestyle

## Related Issue:
- https://github.com/owncloud/updater/issues/582

### CODE REVIEW

File: vendor-bin/php_codesniffer/composer.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (phil-davis) [inline vendor-bin/php_codesniffer/composer.json:1]: We can remove this - the `PHP_CODESNIFFER` standards are ones that we set within QA team for the Behat acceptance test code. They do not apply to other test code.

File: vendor-bin/behat/composer.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (phil-davis) [inline vendor-bin/behat/composer.json:1]: This can be deleted. We do not have Behat acceptance tests in this repo.

File: Makefile [hunk 3]
```diff
@@ -102,6 +104,12 @@ test-php-unit-dbg: vendor/bin/phpunit
 	make
 	$(PHPUNITDBG) ./vendor/bin/phpunit --configuration ./src/Tests/phpunit.xml --testsuite 'ownCloud - Standalone Updater Tests'
 
+.PHONY: test-php-style
+test-php-style:            ## Run php-cs-fixer and check owncloud code-style
+test-php-style: vendor-bin/owncloud-codestyle/vendor vendor-bin/php_codesniffer/vendor
+	$(PHP_CS_FIXER) fix -v --diff --diff-format udiff --allow-risky yes --dry-run
+	$(PHP_CODESNIFFER) --runtime-set ignore_warnings_on_exit --standard=phpcs.xml src/Tests
```
Comment (phil-davis) [inline Makefile:111]: Delete - `PHP_CODESNIFFER` is not needed in this repo.

File: Makefile [hunk 4]
```diff
@@ -17,6 +17,8 @@ dist_dir=$(build_dir)/dist
 COMPOSER_BIN=$(build_dir)/composer.phar
 BOWER=$(build_dir)/node_modules/bower/bin/bower
 PHPUNITDBG=phpdbg -qrr -d memory_limit=4096M -d zend.enable_gc=0
+PHP_CS_FIXER=php -d zend.enable_gc=0 vendor-bin/owncloud-codestyle/vendor/bin/php-cs-fixer
+PHP_CODESNIFFER=vendor-bin/php_codesniffer/vendor/bin/phpcs
```
Comment (phil-davis) [inline Makefile:21]: Delete - `PHP_CODESNIFFER` is not needed

File: Makefile (hunk 3 above)
Comment (jasson99) [reply Makefile]: fixed

File: Makefile [hunk 5]
```diff
@@ -152,7 +152,6 @@ vendor: composer.lock
 
 vendor/bin/phpunit: composer.lock
 	composer install
-	composer require --dev phpunit/phpunit ^7.5
```
Comment (phil-davis) [inline Makefile:144]: This looks like it has been accidentally deleted. Maybe this is why unit tests are failing?

**Comment by phil-davis** [comment]
@jasson99 this looks good for code style. Let's just do code style in this PR. Then you can do another PR  for `phpstan`  and/or `phan` (those might require real code changes).

Please squash and I will approve.

**Comment by phil-davis** [comment]
@jasson99 please change to a signed commit and push again.

**Pull request merged**